package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "start": "NODE_ENV=production ./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
     "start:dev": "NODE_ENV=dev nodemon --config nodemon.json",
-    "lint": "eslint . --fix --ext .js,.jsx,.ts,.tsx",
+    "lint": "ESLINT_USE_FLAT_CONFIG=false eslint . --fix --ext .js,.jsx,.ts,.tsx",
```

Comment:
Eslint v9 introduces new default config format eslint.config.js for now I've set ESLINT_USE_FLAT_CONFIG to false so it can use the old one.